### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Muhtasham**
fixed dataQA link issue also added few weak supervison labeling tools

### CODE REVIEW

**Comment by Muhtasham** [comment]
@strickvl

File: README.md [hunk 1]
```diff
@@ -66,6 +66,17 @@ Slack](https://zenml.io/slack-invite/) and leave us a message!
...
+| [knodle](https://github.com/knodle/knodle) | Knodle (Knowledge-supervised Deep Learning Framework) ![](https://img.shields.io/github/stars/knodle/knodle?color=white&label=%E2%AD%90%EF%B8%8F&style=plastic)| Apache-2 |
+| [skweak](https://github.com/NorskRegnesentral/skweak) | skweak: Weak supervision for NLP ![](https://img.shields.io/github/stars/NorskRegnesentral/skweak?color=white&label=%E2%AD%90%EF%B8%8F&style=plastic)| MIT |
+| [interactive-weak-supervision](https://github.com/benbo/interactive-weak-supervision) | Interactive Weak Supervision: Learning Useful Heuristics for Data Labeling ![](https://img.shields.io/github/stars/benbo/interactive-weak-supervision?color=white&label=%E2%AD%90%EF%B8%8F&style=plastic)| MIT | 
+| [NPLM](https://github.com/BatsResearch/nplm) | Noisy Partial Label Model(NPLM) ![](https://img.shields.io/github/stars/BatsResearch/nplm?color=white&label=%E2%AD%90%EF%B8%8F&style=plastic)| N/A |
+| [bulk](https://github.com/koaning/bulk) | Bulk is a quick developer tool to apply some bulk labels ![](https://img.shields.io/github/stars/koaning/bulk?color=white&label=%E2%AD%90%EF%B8%8F&style=plastic)| MIT |
```
Comment (strickvl) [inline README.md]: These should be alphabetically ordered like the rest of the README is structured. Could you do that? Thanks!

**Review by strickvl — CHANGES_REQUESTED** [review]
Thanks for the submission. I think we just need a reordering.

File: README.md [hunk 2]
```diff
@@ -55,25 +55,26 @@ Slack](https://zenml.io/slack-invite/) and leave us a message!
...
+| [bulk](https://github.com/koaning/bulk) | Bulk is a quick developer tool to apply some bulk labels ![](https://img.shields.io/github/stars/koaning/bulk?color=white&label=%E2%AD%90%EF%B8%8F&style=plastic)| MIT | 
 | [CoreNLP](https://github.com/stanfordnlp/CoreNLP) | A Java suite of core NLP tools ![](https://img.shields.io/github/stars/stanfordnlp/CoreNLP?color=white&label=%E2%AD%90%EF%B8%8F&style=plastic) | GPL-3 |
 | [DataQA](https://github.com/dataqa/nlp-labelling) | Labeling platform for text using weak supervision ![](https://img.shields.io/github/stars/dataqa/nlp-labelling?color=white&label=%E2%AD%90%EF%B8%8F&style=plastic) | GPL-3 |
 | [doccano](https://github.com/doccano/doccano) | An open source text annotation tool supporting text classification, sequence labeling and sequence to sequence tasks ![](https://img.shields.io/github/stars/doccano/doccano?color=white&label=%E2%AD%90%EF%B8%8F&style=plastic) | MIT |
+| [FlyingSquid](https://github.com/HazyResearch/flyingsquid) | More Interactive Weak Supervision with FlyingSquid ![](https://img.shields.io/github/stars/HazyResearch/flyingsquid?color=white&label=%E2%AD%90%EF%B8%8F&style=plastic)| Apache-2 |
```
Comment (strickvl) [inline README.md]:
```suggestion
```

This library is not actively maintained.

File: README.md [hunk 3]
```diff
@@ -55,25 +55,26 @@ Slack](https://zenml.io/slack-invite/) and leave us a message!
...
 | [FLAT - FoLiA Linguistic Annotation Tool](https://github.com/proycon/flat) | A web-based linguistic annotation environment based around the FoLiA format, an XML-based format for linguistic annotation ![](https://img.shields.io/github/stars/proycon/flat?color=white&label=%E2%AD%90%EF%B8%8F&style=plastic) | GPL-3 |
 | [INCEpTION](https://github.com/inception-project/inception) | A semantic annotation platform offering intelligent annotation assistance and knowledge management ![](https://img.shields.io/github/stars/inception-project/inception?color=white&label=%E2%AD%90%EF%B8%8F&style=plastic) | Apache-2 |
+| [interactive-weak-supervision](https://github.com/benbo/interactive-weak-supervision) | Interactive Weak Supervision: Learning Useful Heuristics for Data Labeling ![](https://img.shields.io/github/stars/benbo/interactive-weak-supervision?color=white&label=%E2%AD%90%EF%B8%8F&style=plastic)| MIT |
```
Comment (strickvl) [inline README.md]:
```suggestion
```

This library is not actively maintained.

**Review by strickvl — CHANGES_REQUESTED** [review]
Thanks for re-ordering. I've taken two out as they don't appear to be actively maintained, but otherwise this looks almost ready to merge in.

**Comment by strickvl** [comment]
Thanks for the PR, @Muhtasham ! 🙌

**Pull request merged**